web-ui/src/components/search-results/SearchBirthdayAnniversaryResults.jsx

```diff
@@ -15,26 +15,25 @@ import {
 
 const SearchBirthdayAnniversaryResults = ({
   hasSearched,
-  searchBirthdayResults,
-  searchAnniversaryResults,
+  results,
   anniversary,
   birthday,
 }) => {
-  searchAnniversaryResults.sort((a, b) => {
-    return a.tenure - b.tenure;
-  });
-
-  searchBirthdayResults.sort((a, b) => {
-    const adate = new Date(a.birthDay);
-    const bdate = new Date(b.birthDay);
-    return adate - bdate;
-  });
+  if (anniversary) {
```

Comment:
We sort the array differently based on whether the items represent anniversaries or birthdays.